lang/nl.js

```diff
@@ -7,6 +7,12 @@ export default {
 	"components.calendar.selected": "Geselecteerd.",
 	"components.calendar.show": "{month} weergeven",
 	"components.dialog.close": "Dit dialoogvenster sluiten",
+	"components.filter.loading": "Loading filters",
+	"components.filter.filterCount": "{number, select, max {99+} all {All} other {{number}}",
```

Comment:
Got this from Google Translate but it matches German so seems right?

```suggestion
	"components.filter.filterCount": "{number, select, max {99+} all {Alle} other {{number}}",
```